cdhweb/pages/templates/cdhpages/home_page.html

```diff
@@ -18,12 +18,17 @@
 {% block main %}
 
 {# display editable page content; wrapped for formatting reasons #}
-<section>
+<section class="textcontent">
     <div class="richtext">
     {% include_block page.body %}
     </div>
 </section>
 
+{# featured pages (about & consult) — only render if pages are available #}
+{% if about and consult %}
```

Comment:
why not if only one is available? display doesn't handle just one?

Reply:
there's no design for that, is there? what would we do in such a case?

Reply:
I don't know! I guess you could center it, but agree we don't have a design for that and don't need to support it right now